python/ray/dag/compiled_dag_node.py

```diff
@@ -560,6 +560,7 @@ def __init__(
         asyncio_max_queue_size: Optional[int] = None,
         max_buffered_results: Optional[int] = None,
         max_inflight_executions: Optional[int] = None,
+        custom_nccl_group: Optional[GPUCommunicator] = None,
```

Comment:
Let's rename this to `default_nccl_group`. This seems more specific and accurate.

Reply:
Done.